src/main/resources/com/microsoft/jenkins/containeragents/aci/AciContainerTemplate/config.jelly

```diff
@@ -49,7 +49,8 @@
                                     deleteCaption="${%Delete_Volume}"/>
     </f:entry>
 
-    <ui:dropdownList selectedName="launchMethodType" title="${%Launch_Method}" name="launchMethodTypeContent">
+    <!-- TODO this doesn't seem to be written correctly, when page is refreshed inbound agent is always displayed -->
```

Comment:
swapping to ssh doesn't seem to work properly, but appears to be broken before this change